### PR TITLE
New ART check

### PR DESCRIPTION
**Pull request by hathin**
Added docs entry for new ART check, Guest user access

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/managed-desktop/get-ready/readiness-assessment-fix.md [hunk 1]
```diff
@@ -283,6 +283,14 @@ Microsoft Managed Desktop recommends adjusting guest invitation settings, since
 
 **Member users and users assigned to specific admin roles can invite guest users including guests with member permissions** should be enabled. For steps, see [Prerequisites for guest accounts](/microsoft-365/managed-desktop/get-ready/guest-accounts).
 
+### Guest user access
+
+Microsoft Managed Desktop recommends adjusting guest user access, since the default setting allows all guest users in your directory to have the same access as members.
+
+**Advisory**
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/get-ready/readiness-assessment-fix.md:290]:
```suggestion
#### Advisory
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @hathin, kindly commit the suggested change if you agree with it. Many thanks.

File: microsoft-365/managed-desktop/get-ready/readiness-assessment-fix.md (hunk 1 above)
Comment (hathin) [reply microsoft-365/managed-desktop/get-ready/readiness-assessment-fix.md:290]: This does not follow the pattern for the other checks. Advisory is not a header but instead should just be bold. I believe **Advisory** is the correct way to show this

**Review by hathin — COMMENTED** [review]
This does not follow the pattern for the other checks. Advisory is not a header but instead should just be bold. I believe **Advisory** is the correct way to show this

File: microsoft-365/managed-desktop/get-ready/readiness-assessment-fix.md (hunk 1 above)
Comment (JohanFreelancer9) [reply microsoft-365/managed-desktop/get-ready/readiness-assessment-fix.md:290]: Thank you for your response. I take your point. Please ignore my suggestion. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@jaimeo Please merge this PR. Thanks

**Pull request merged**